### ISSUE TITLE
Checkbox Group A11y Issues

### ISSUE THREAD

**Issue by CillianHPE** [grommet/hpe-design-system#4894]
### Link to problem

https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=7OzniU&nav=MTVfezRBQUIzRURCLTI4MEUtNDY1QS04RUFFLTUwMTc3MTU4QkJGOX0

### Current problem

Relating to WCAG Rule 1.3.1
- Users should be notified how many items are within a checkbox group
- Users should be notified about which item within that group they are currently focused on.

Relating to WCAG rule 2.1.1 Keyboard
- Users shoul be also able to use arrow keys to navigate within the checkbox group

Relating to WCAG rule 2.5.5
- Checkbox does not meet 44x44px target size.

**Comment by CillianHPE** [comment]
Closing this issue as it was created in the wrong repo.
